composeApp/src/commonMain/kotlin/world/respect/app/app/AppScreenList.kt

```diff
@@ -0,0 +1,26 @@
+package world.respect.app.app
+
+import kotlinx.serialization.Serializable
+
+sealed interface AppDestination
+
+@Serializable
+object AppLauncher: AppDestination
+
+@Serializable
+object Assignment: AppDestination
+@Serializable
+object Clazz: AppDestination
+@Serializable
+object Report: AppDestination
+@Serializable
+object AppList: AppDestination
+@Serializable
+object EnterLink: AppDestination
+@Serializable
+data class AppsDetail(val manifestUrl: String): AppDestination
+@Serializable
+data class LessonList(val manifestUrl : String): AppDestination
```

Comment:
Wrong variable named - this is not the manifestUrl - could be just url or opdsUrl